lib/issue_exporter/import.rb

```diff
@@ -0,0 +1,87 @@
+require 'net/http'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.